wiki/Announcements/Contests/Monthly_Fanart_Contest/A_very_osu!_halloween/en.md

```diff
@@ -0,0 +1,95 @@
+# A very osu! Halloween
+
+## Public Picks
+
+### First Place
+
+- by [DumplingYumYum](https://osu.ppy.sh/u/1715930)
+
+![Entry](img/1p.jpg)
+
+### Second Place
+
+- by [CosmicTea](https://osu.ppy.sh/u/5050769)
+
+![Entry](img/2p.jpg)
+
+### Third Place
+
+- by [RedViolet](https://osu.ppy.sh/u/2221040)
+
+![Entry](img/3p.jpg)
+
+### Fourth Place
+
+- by [JMC](https://osu.ppy.sh/u/774010)
+
+![Entry](img/4p.jpg)
+
+### Fifth Place
+
+- by [UUL](https://osu.ppy.sh/u/3769802)
+
+![Entry](img/5p.jpg)
+
+## Staff Pick
+
+- by [Anny](https://osu.ppy.sh/u/249094)
+
+![Entry](img/1s.jpg)
+
+### Runner-up (No Particular Order)
+
+<!-- Runner-up below and 3rd Public Pick were the same -->
```

Comment:
I wouldn't, it will be to remind me (and future editors) that `2s.jpg` doesn't exist since it is the same image as `3p.jpg`.

While on this topic, is there a hidden wiki standard that I overlooked that says we should get rid of all comments? (even those that may be helpful?) If not, want to add your input about this?